modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/service/internal/scheduler/test/WorkflowMetricsSLADefinitionTransformerSchedulerJobConfigurationTest.java

```diff
@@ -169,6 +173,54 @@ public void testTransform2() throws Exception {
 			workflowMetricsSLADefinitionVersion.getStatus());
 	}
 
+	@Test
+	public void testWorkflowDefinitionSLADoesNotExceedIndexSearchLimit()
+		throws Exception {
+
+		String processIndex =
+			_indexNameBuilder.getIndexName(workflowDefinition.getCompanyId()) +
+				WorkflowMetricsIndexNameConstants.SUFFIX_PROCESS;
+
+		UpdateIndexSettingsIndexRequest updateIndexSettingsRequest =
+			new UpdateIndexSettingsIndexRequest(processIndex);
+
+		updateIndexSettingsRequest.setSettings(
+			"{\"index\":{\"max_result_window\":\"2000\"}}");
+
+		_searchEngineAdapter.execute(updateIndexSettingsRequest);
+
+		String originalLimit = PropsUtil.get(PropsKeys.INDEX_SEARCH_LIMIT);
```

Comment:
Can you rename this variable to `originalIndexSearchLimit` or just `indexSearchLimit`?